crates/unleash-edge-http-client/src/lib.rs

```diff
@@ -634,6 +632,7 @@ impl UnleashClient {
                 )
             })?;
 
+        println!("Received heartbeat response: {:?}", response);
```

Comment:
I assume this was just for debug purposes and should be cleaned up? Otherwise make it a `debug!` maybe?


```suggestion

```